cc7_lcg95/Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG branch
+FROM gitlab-registry.cern.ch/acts/machines/cc7:$branch
+
+RUN yum install -y http://linuxsoft.cern.ch/wlcg/centos7/x86_64/wlcg-repo-1.0.0-1.el7.noarch.rpm \
+    && yum install -y HEP_OSlibs \
+    && yum-config-manager --add-repo https://lcgpackages.web.cern.ch/lcgpackages/rpms/etc/yum.repos.d/lcg.repo \
+    && yum clean all
+
+ENV LCG_PREFIX=95apython3
+ENV PLATFORM=x86_64-centos7-gcc8-opt
+
+RUN yum install -y \
+      LCG_${LCG_PREFIX}_ROOT_6.16.00_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_CMake_3.11.1_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_eigen_3.3.7_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_pythia8_240_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_cmaketools_1.8_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_lcgenv_1.3.8_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_DD4hep_01_10_${PLATFORM//-/_}.noarch \
+      LCG_${LCG_PREFIX}_Geant4_10.05_${PLATFORM//-/_}.noarch \
+    && yum clean all
+
+RUN cd /opt/lcg \
+    && git clone https://gitlab.cern.ch/adye/lcgcmake.git \
```

Comment:
Do this full repository need to be cloned or would the `create_lcg_view` script be sufficient. If so, we can just ship it with the repository.

Reply:
Ideally I don't want to maintain a copy. I concede that pointing to Tim's repo here is not really great either, I'll ask him if he thinks his patch could be upstreamed.

Reply:
Updated to the upstream repo.